sdk/deepbook/vitest.config.ts

```diff
@@ -5,8 +5,8 @@ import { defineConfig } from 'vitest/config';
 
 export default defineConfig({
 	test: {
-		minThreads: 1,
```

Comment:
These options don't exist anymore, I suspect that running too many tests in parallel is whats causing the execution timeouts and flakiness